packages/ui/ProgressBar/ProgressBar.module.less

```diff
@@ -10,11 +10,18 @@
 	direction: ltr;
 
 	.bar,
+	.separator,
 	.fill,
 	.load {
 		position: absolute;
 	}
 
+	.separator {
```

Comment:
For reference, the variable for the progress anchor is `var(--ui-progressbar-proportion-start)`

Reply:
It seems that `--ui-progressbar-proportion-start` does not synchronized `progressAnchor` value. So, I add `--ui-progressbar-progress-anchor` for handle it.